src/main/resources/org/jenkinsci/plugins/ewm/strategies/MostUsableSpaceStrategy/help-estimatedWorkspaceSize.html

```diff
@@ -0,0 +1,31 @@
+<div>
+    <p>
+        Optional parameter.
+        It indicates the estimated size that the workspace may have.
+        It has to be in KB.
+        This can influence the Disk allocation strategy used.
+    </p>
+    <p>
+        Examples:
+    </p>
+    <p>
+        1. Let's suppose that you choose the strategy that selects the disk with the most usable space, and you have
```

Comment:
You could reduce duplication by writing help in Jelly instead of HTML and storing Jelly on the base class level.
Storing on the upper class level should work for HTML as well